spec/models/asp/payment_request_spec.rb

```diff
@@ -148,19 +148,54 @@
 
     context "when the payment request is in 'incomplete' state with the abrogation specific error message" do
       it "returns true" do
-        expect(p_r_incomplete_for_abrogation.eligible_for_auto_retry?).to be true
+        expect(p_r_incomplete_for_abrogation.eligible_for_incomplete_retry?).to be true
       end
     end
 
     context "when the payment request is in 'incomplete' state with the missing DA specific error message" do
       it "returns true" do
-        expect(p_r_incomplete_for_missing_da.eligible_for_auto_retry?).to be true
+        expect(p_r_incomplete_for_missing_da.eligible_for_incomplete_retry?).to be true
       end
     end
 
     context "when the payment request is not in 'incomplete' state" do
       it "returns false" do
-        expect(p_r_ready.eligible_for_auto_retry?).to be false
+        expect(p_r_ready.eligible_for_incomplete_retry?).to be false
+      end
+    end
+  end
+
+  describe "eligible_for_rejected_or_unpaid_auto_retry?" do
+    let(:p_r_rejected) { create(:asp_payment_request, :rejected) }
+    let(:p_r_rejected_rib) do
+      create(:asp_payment_request, :rejected, reason: "Test d'une raison de blocage d'un paiement bancaire")
+    end
+    let(:p_r_unpaid) { create(:asp_payment_request, :unpaid) }
+    let(:p_r_unpaid_rib) do
+      create(:asp_payment_request, :unpaid, reason: "Test d'une raison de blocage d'un paiement bancaire")
```

Comment:
Ta raison doit matcher le eligible for auto retry, donc contenir "[RIB BIC PAIEMENT]"